### DISCUSSION TITLE
How to import from OpenAPI V3 URL?

### DISCUSSION THREAD

**Discussion by cloudenum** [usebruno/bruno#3463] [Q&A] [answered]
The [docs](https://docs.usebruno.com/get-started/import-export-data/import-collections) says you can import via OpenAPI V3 URL. But I can't find that option in the latest version (1.34.2). Does anyone knows where or is this feature only available on the Golden Edition?

**Comment by rreyn-bruno** [comment, marked as answer]
Hi @cloudenum - yes, currently importing OAS V3 via URL is a premium feature.

**Reply by cloudenum** [reply]
Ah I see
The docs needs to be updated then
